docs/conf.py

```diff
@@ -37,6 +37,7 @@
 # ones.
 extensions = [ 
   'sphinx_js',
+  'recommonmark',
```

Comment:
Since it's been converted to rst, there's no need for the changes in this file.

Reply:
These changes should have been removed